### DISCUSSION TITLE
Is there a way to add components contained in an module to the page

### DISCUSSION THREAD

**Discussion by vnetonline** [oqtane/oqtane.framework#3000] [Q&A] [answered]
Let’s say for example we had a blogging module which contains a 

- calendar view 
- List of categories
- list of tags 
- a list of blogs 

Can these three components each with its own index.razor and action pages can they be added to the page when the module is added from the Admin Dashboard all at once 

I think this feature was available in DNN

**Comment by leigh-pointer** [comment, marked as answer]
Yes i have been doing this for a long time now. Simply  create a new view in a new name space and its unique ModuleInfo class.

**Comment by sbwalker** [comment]
@vnetonline you are referring to the DesktopModule "grouping" in DNN. Oqtane does not support this feature - and I am not planning on adding it to the roadmap as it added extra complexity which I want to avoid.
